android/gradle.properties

```diff
@@ -1,5 +1,5 @@
 android.useAndroidX=true
-Purchases_kotlinVersion=1.4.0
+Purchases_kotlinVersion=1.6.21
```

Comment:
We should get this in master too, right?

Reply:
Yup! Merged in main 😄

Reply:
Errr main 🤦🏻‍♂️